examples/alerts/alerts.md

```diff
@@ -1,6 +1,19 @@
 # Alerts
 
 Here are some example alerts configured for Kubernetes environment.
+The whole alert rule config file is as following:
+```
```

Comment:
New line has to before the code snippet (:

Reply:
1. Newline is not addressed. Markdown requires this. 
2. The reason why it would be better on the ruler, as those are alerts/rules *specifically* in Thanos Ruler. Examples of alerts here are mainly to operate Thanos, but it does not mean you have to set up alerts on Thanos Ruler straight away! (:  
3. Let's mention recording rules as well (: